audit/projects/k8s-staging-artifact-promoter/services/compute/project-info.json

```diff
@@ -1,171 +0,0 @@
-{
-  "commonInstanceMetadata": {
-    "kind": "compute#metadata"
-  },
-  "creationTimestamp": "2020-05-06T21:55:38.693-07:00",
-  "defaultNetworkTier": "PREMIUM",
-  "defaultServiceAccount": "675573440409-compute@developer.gserviceaccount.com",
-  "id": "1279192389912504869",
-  "kind": "compute#project",
```

Comment:
Removal of compute project-info is expected due to disablement of the compute service (see other review comments); this service should not have been enabled in the first place